src/core/lib/security/security_connector/ssl_utils.cc

```diff
@@ -43,8 +43,10 @@
 #ifndef INSTALL_PREFIX
 static const char* installed_roots_path = "/usr/share/grpc/roots.pem";
 #else
+#define QUOTE(x) #x
+#define EXPAND_AND_QUOTE(x) QUOTE(x)
 static const char* installed_roots_path =
-    INSTALL_PREFIX "/share/grpc/roots.pem";
+    EXPAND_AND_QUOTE(INSTALL_PREFIX) "/share/grpc/roots.pem";
```

Comment:
@ZhenLian @jiangtaoli2016  this part of the fix might be onto something. What is the expected use for  the INSTALL_PREFIX define?
(btw I didn't know it existed. Also, is the INSTALL_PREFIX define something normally supported by C++ applications? If not, the "INSTALL_PREFIX" define's name seems not specific enough (AFAIK we're only using it for figuring out the roots.pem path?).